Polish/przygoda/jedzenie/wiedzmin/wiedzmin.md

```diff
@@ -0,0 +1,2 @@
+Teleportujesz się na skwer przed ulubioną księgarnią komputerową skrywającą staroświeckie papierowe woluminy wszechwiedzy. Skwer jest rozgrzany popołudniowym słońcem lipcowego skwaru. Twój wzrok automatycznie kieruje się ku nogom dwóch przechodzących dziewcząt. Kobiety mijają cię, a ty odwracasz się za nimi. Podnosisz wzrok w kierunku kołyszących się pup. Odpędzasz od siebie nieczyste myśli, odwracasz się na pięcie i wchodzisz się do księgarni. Sprzedawca, cybernetyczny, zasuszony hipernerd wita cię tymi słowami:
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.